lib/exq_scheduler/storage.ex

```diff
@@ -30,13 +31,17 @@ defmodule ExqScheduler.Storage do
 
   def persist_schedule(schedule, storage_opts) do
     val = Schedule.encode(schedule)
-    _ = Redis.hset(storage_opts.redis, build_schedules_key(storage_opts), schedule.name, val)
+    _ = Redis.hset(storage_opts.lib,
+      storage_opts.redis,
```

Comment:
I guess we could pass the whole opts to Redis module. `module` & `name` might be better than `redis` & `lib`